pages/clusters/queue_metrics.md

```diff
@@ -0,0 +1,24 @@
+# Queue metrics in clusters
+
+Queue metrics show the most important statistics to help you optimize your agent setup and monitor a cluster's performance. These statistics are updated on the page every 10 seconds.
+
+<%= image "cluster-queue-metrics.png", alt: "Screenshot of the queue metrics panel" %>
+
+## Agents panel
+
+_Agents Connected_ is the number of agents connected to the cluster queue. The circular chart represents the fraction of agents that are busy working on jobs compared to those that are idle and ready for a job. Hovering over the chart shows the _Agent Utilization_ panel, which displays the percentage values for each chart component.
+
+For agent utilization, agents are considered busy if they have a job ID assigned.
+
+<%= image "cluster-queue-metrics-agent-utilization.png", width: 530/2, height: 434/2, alt: "Screenshot of the agent utilization panel" %>
+
+## Jobs panel
+
+Jobs Running and Jobs Waiting shows the number of jobs assigned to an agent, and the number of jobs
+not yet assigned to an agent respectively. Jobs Running are any jobs for this cluster queue which
+are in a state of RUNNING, CANCELING, TIMING_OUT, ACCEPTED, or ASSIGNED. Jobs Waiting are any jobs
+for this cluster queue which are in a state of SCHEDULED.
```

Comment:
Sorry, I made changes here in the last round, but it looks like the comment didn't save. I think we should split this up to only talk about one field at a time. I've also added a list to help break up the text and make it a bit more scannable.

```suggestion
_Jobs Running_ shows the number of jobs assigned to agents. These are any jobs in the cluster queue in the following states:

- `RUNNING`
- `CANCELING`
- `TIMING_OUT`
- `ACCEPTED`
- `ASSIGNED`

_Jobs Waiting_ shows the number of jobs not yet assigned to an agent. These are any jobs for the cluster queue in the `SCHEDULED` state.
```